lms/djangoapps/courseware/module_render.py

```diff
@@ -157,6 +157,7 @@ def toc_for_course(request, course, active_chapter, active_section, field_data_c
         for chapter in chapters:
             # Only show required content, if there is required content
             # chapter.hide_from_toc is read-only (boo)
+            name_without_spaces = chapter.display_name_with_default.replace(" ", "-").replace(":", "")
```

Comment:
@andy-armstrong I would tag @ahsan-ul-haq on the functional bits. I only handled styling for this piece of work.

I will, however, get you a sandbox spun up!